managed/utils/interceptors/interceptors.go

```diff
@@ -75,8 +76,7 @@ func logRequest(l *logrus.Entry, prefix string, f func() error) (err error) {
 		}
 	}()
 
-	err = f()
-	return //nolint:nakedret
+	return f()
```

Comment:
we still need `err = f()` here for logic in `defer`

Reply:
please check other places where you removed `err = .*` to make sure that `err` isn't used in `defer` function

Reply:
Yeah right. Anyway PR is not ready yet.